### PR TITLE
feat(Schematics): use ofType operator function instead of Actions#ofType

### PR DESCRIPTION
**Pull request by kouMatsumoto**
I suppose that we should use ofType operator function instead of Actions#ofType in generated files by schematics.
Please consider this change, thank you!

### CODE REVIEW

File: modules/schematics/src/effect/index.spec.ts [hunk 1]
```diff
@@ -50,6 +50,44 @@ describe('Effect Schematic', () => {
...
+    );
+    expect(content).toMatch(
+      /import { Actions, Effect, ofType } from '@ngrx\/effects';/
+    );
+    expect(content).toMatch(
+      /import { LoadFoos, FooActionTypes } from '\.\/foo.actions';/
+    );
+    expect(content).toMatch(/export class FooEffects/);
+    expect(content).toMatch(
+      /effect\$ = this\.actions\$.pipe\(ofType<LoadFoos>\(FooActionTypes\.LoadFoos\)\);/
```
Comment (tdeschryver) [inline modules/schematics/src/effect/index.spec.ts]: Maybe change `effects$` to be `loadFoos$`?

Comment (kouMatsumoto) [reply modules/schematics/src/effect/index.spec.ts]: Thank you for your comment!
I renamed "effect$" to "loadFoos$".
It's more suitable in this case.

File: modules/schematics/src/effect/index.spec.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ describe('Effect Schematic', () => {
     appTree = createWorkspace(schematicRunner, appTree);
   });
 
-  it('should create an effect', () => {
+  it('should create an effect file and its spec file', () => {
```
Comment (tdeschryver) [inline modules/schematics/src/effect/index.spec.ts]: This should be `should create an effect with a spec file`.

File: modules/schematics/src/effect/index.spec.ts [hunk 3]
```diff
@@ -50,6 +50,44 @@ describe('Effect Schematic', () => {
     ).toBeGreaterThanOrEqual(0);
   });
 
+  it('should create an effect class which has a "loadFoos$" property if feature is set', () => {
```
Comment (tdeschryver) [inline modules/schematics/src/effect/index.spec.ts]: This should be `should create an effect that describes a source of actions within a feature`.

File: modules/schematics/src/effect/index.spec.ts [hunk 4]
```diff
@@ -50,6 +50,44 @@ describe('Effect Schematic', () => {
...
+    expect(content).toMatch(
+      /import { LoadFoos, FooActionTypes } from '\.\/foo.actions';/
+    );
+    expect(content).toMatch(/export class FooEffects/);
+    expect(content).toMatch(
+      /loadFoos\$ = this\.actions\$.pipe\(ofType<LoadFoos>\(FooActionTypes\.LoadFoos\)\);/
+    );
+  });
+
+  it('should create an effect class which does not have a "loadFoos$" property if root is set', () => {
```
Comment (tdeschryver) [inline modules/schematics/src/effect/index.spec.ts]: This should be `should create an effect that does not define a source of actions within the root`.
And maybe move the test a bit down? (after the default tests)

**Comment by MatsumotoKou** [comment]
@tdeschryver 
Thank you for review and explaining!
I updated it, please continue your checks.

File: modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
+<% if (feature) { %>import { Load<%= classify(name) %>s, <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
 
 @Injectable()
 export class <%= classify(name) %>Effects {
-<% if(feature) { %>
+<% if (feature) { %>
   @Effect()
-  effect$ = this.actions$.ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>s);
+  loadFoos$ = this.actions$.pipe(ofType<Load<%= classify(name) %>s>(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>s));
 <% } %>
```
Comment (brandonroberts) [inline modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts]: Remove the `<Load<%= classify(name) %>s>`

**Comment by kouMatsumoto** [comment]
@brandonroberts 
Thank you for the pointing out!
I update it, please continue your review.

**Pull request merged**